### ISSUE TITLE
Publication ticket: Batch 2 of new presidential guidance pages

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#3205]
### Summary

**What we are after:**
We are creating two new buckets with related pages within Help for candidates and committees geared to presidential candidates. One bucket concerns public funding and the other concerns the transition and inauguration process.

### Related issues
- #3046 (Circulation ticket and includes links to all drafting tickets)

### Completion criteria

After stakeholders return comments (due November 29) on draft new pages:
For each page will need to:
- [x] Combine the page edits
- [x] Address any comments or edits that need working out
- [x] Create clean working copy of each separate page for use in making the pages
~- [ ] Upload the Financial Control and Compliance Manual for General Election Candidates Receiving Public Funding~ (Audit Division request; they need to find it, so I'll take it out of this ticket and just upload for them when they do.)
~- [ ] Recirculate if extensive edits from multiple places~
- [x] Create Collection page for public funding bucket
- [x] Create Collection page for transition/inauguration bucket
- [x] Create new pages (custom or resource if divided into sections) as child pages of these new collection pages (note: this is a task that Info Division specialists can help with)

Public funding bucket child pages to create:
•	Establishing eligibility to receive presidential primary matching fund payments - RECIRCULATED AND APPROVED
•	Using presidential primary matching funds and paying for qualified campaign expenses
•	Complying with primary spending limits
•	Receiving a grant for the general election
•	Using the general election public funds grant and paying for qualified campaign expenses
•	Raising funds for legal and accounting expenses (GELAC fund)
•	Public funding and vice presidential candidates
•	Providing transportation and services to media, Secret Service and national security staff
•	Additional recordkeeping for publicly funded candidates
•	Complying with audit and review requirements
•	Debts owed by publicly funded candidates
•	Winding down publicly funded presidential candidate committees
Presidential spending limits for 2016.

Transition/inauguration bucket child pages to create:
•	Funding the presidential transition
•	Funding inaugural committee activities
•	Inaugural committee reports - DONE in earlier sprint.

- [x] Second pair of eyes
- [x]  Publish the child pages
- [x] Add the links to the collection pages and publish the collection pages
 
### Future work
- Front end ticket to update the Guides page and the H4CC landing page to include links to the new buckets: #3381 
- Adding the new glossary term links: #3425 
- Updating cross-references and "read next" throughout H4CC to include these pages: #3352 
-  Add any needed redirects (e.g., delegate page) to redirects spreadsheet

**Comment by dorothyyeager** [comment]
We've heard back from Audit, OGC's compliance advice team, RAD. Still waiting on Press and Policy. After the holiday, I'll check in with those offices and see where we are.

**Comment by dorothyyeager** [comment]
We've heard back from Policy (their edits were combined with the compliance advice team's) and Press (no edits from them). So now we're ready to work on this ticket!

**Comment by dorothyyeager** [comment]
I've combined the edits we received from stakeholders and identified a few spots where we'll need to make an editorial decision. Overall, though, the edits were pretty minor for the most part and we're ready to make some of the pages. Unfortunately the collection page we need to make first is one of the editorial decisions, so that's a tiny blocker. Will talk with content team tomorrow about next steps
.

**Comment by dorothyyeager** [comment]
I was able to create the collection pages and save as draft, with placeholders for the text we'll use for the public funding one, since that needs some edits.

George in Info is reviewing the comments by OGC, Audit and RAD and we'll either go with them or perhaps circle back on a few. There were no comments on the transition and inaugural bucket pages.

Audit is looking for the PDF file of the general election manual they want to upload and link in these pages.

**Comment by dorothyyeager** [comment]
We've gotten back to OGC and Audit on one paragraph they both commented on and reconciled those edits. OGC is also OK with us not adding too much text to the collection page. I went ahead and filled in the placeholder on draft public funding collection page.

So with the language now approved and collection pages created in draft form, we can move on to the next step - Accepting the edits and making clean text versions of the new pages.

**Comment by dorothyyeager** [comment]
I'm working on making the clean working files for each new page. A specialist from the Info Division will assist me in creating the pages when the working files are ready.

**Comment by dorothyyeager** [comment]
@kathycarothers will work on the transition bucket and I'll work with Don in Information Division on the public funding bucket.

**Comment by kathycarothers** [comment]
@dorothyyeager transition bucket is done.

**Comment by dorothyyeager** [comment]
Transition pages are basically ready to go! As is the 2016 spending limits page in #3416. Work continues on the public funding bucket pages. We're waiting to hear back from OGC on the glossary.

**Comment by dorothyyeager** [comment]
The pages have been created! We are now in the second pair of eyes stage.

**Comment by dorothyyeager** [comment]
The pages have been published! Work on this issue is complete and we'll move on to updating the glossary (tickets #3221 and #3425) and adding the bucket links to the Guides page (#3381)
